qoqo_qiskit/qoqo_qiskit/backend/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright © 2023 HQS Quantum Simulations GmbH.
+"""Qoqo-qiskit backend package for compatibility and simulation purposes."""
+
+
+from qoqo_qiskit.backend.backend import QoqoQiskitBackend
```

Comment:
Should this be named QiskitBackend, to conform to the naming convention of our other backends?

Reply:
I think leaving it like this is actually good, because we would probably use it in an environment where there are also actual qiskit backends (as in qiskit providers) so that makes the naming quite clear.